test/liquid_voting/delegations_test.exs

```diff
@@ -135,7 +135,7 @@ defmodule LiquidVoting.DelegationsTest do
       assert {:ok, %Delegation{} = delegation} = Delegations.create_delegation(args)
     end
 
-    test "create_delegation/1 with duplicate proposal-specific data returns error changeset",
+    test "create_delegation/1 with duplicate proposal-specific data returns the original delegation",
```

Comment:
Since the new clause `create_delegation(%{delegator_id: _, delegate_id: _} = attrs)` passes the received `attrs` to `upsert_delegation/1`, an attempted creation of a duplicate delegation will avoid violating the unique constraint `:org_delegator_delegate_proposal` and the exisiting delegation will be returned (as intended).